### ISSUE TITLE
The On-Screen keyboard is behind the console output

### ISSUE THREAD

**Issue by mrfloydst** [shorepine/tulipcc#387]
Console output is written over the onscreen keyboard, which is kind of distracting.

**Comment by bwhitman** [comment]
Yes, annoying and maybe (for now) hard for us to fix -- the console is drawn at the very last moment before rendering out the screen, for performance reasons.  One sort-of-solution is to set the height of the console to half the screen with the OSK is running? We could probably set that up in `ui.py`.
